tests/integration/fixtures/mini_sentry.py

```diff
@@ -308,17 +308,25 @@ def get_project_config():
         if relay_id not in authenticated_relays:
             abort(403, "relay not registered")
 
+        response = {}
         configs = {}
+        global_key = {}
         pending = []
+
         version = flask_request.args.get("version")
         if version in [None, "1"]:
-            for project_id in flask_request.json["projects"]:
+            for project_id in flask_request.json.get("projects", []):
                 project_config = sentry.project_configs[int(project_id)]
                 if is_trusted(relay_id, project_config):
                     configs[project_id] = project_config
 
-        elif version in ["2", "3"]:
-            for public_key in flask_request.json["publicKeys"]:
+        if version in ["2", "3", "4"]:
+            if version == "4":
+                if flask_request.json.get("global") or True:
+                    global_key["measurements"] = {"maxCustomMeasurements": 0}
+                    global_key["metricsConditionalTagging"] = []
```

Comment:
looks silly atm, this will be more meaningful when we make a proper default for globalconfig